napari/components/_viewer/model.py

```diff
@@ -31,16 +31,19 @@ class Viewer:
         These key bindings are executed instead of any layer specific key
         bindings.
     """
-    def __init__(self, title='napari'):
+
+    def __init__(self, title="napari"):
```

Comment:
Is the any way to configure black to use `'` instead of `"`?

Reply:
No as far as i know. This is something `black` has a strong preference. To me also it makes sense, as double `"` is used for strings as a widely popular convention. On top of it, it is consistent with C/OpenCL and so. IMO we are very likely to depend such languages/tools in the long run.

Reply:
yeah - i just found a super long [hacker news post](https://news.ycombinator.com/item?id=17151813) on this one! Let's just say some very strong opinions on either side, and a developer who won't budge.

Reply:
Everybody(including me) has an opinion on everything(especially in python world) :D that's why maybe better to decide according to priorities instead of opinions.

Reply:
btw, good news it is possible to stop `black` on this with: `--skip-string-normalization`

Reply:
yes - very interesting, I just read this thread here https://github.com/python/black/issues/118, and the main developer very reasonably added that skip, which can be achieved with the short-cut `black . -S`